ios/SiftReactNative.m

```diff
@@ -12,15 +12,14 @@ - (dispatch_queue_t)methodQueue
 
 # pragma mark - Configuration
 
-RCT_EXPORT_METHOD(setSiftConfig:(NSString *)accountId beaconKey:(NSString *)beaconKey disallowCollectingLocationData: (BOOL)disallowCollectingLocationData serverUrlFormat:(NSString *)serverUrlFormat allowUsingMotionSensors: (BOOL)allowUsingMotionSensors) {
-    Sift *sift = [Sift sharedInstance];
```

Comment:
This line deletion breaks iOS compilation. Fixed: https://github.com/SiftScience/sift-react-native/pull/13